docs/code-of-conduct.rst

```diff
@@ -0,0 +1,33 @@
+Code of Conduct
+===============
+
```

Comment:
Becky (@yo-bj) suggested that we keep more of the original sentence, in this comment: https://github.com/writethedocs/www/pull/115/files/ef1be4587e35ad99a53575591c92ef8508ad1142#r95433127

I think it's OK to replace it with the basic statement borrowed from the main writethedocs.org page